packages/sensors_plus/sensors_plus/ios/Classes/FPPStreamHandlerPlus.swift

```diff
@@ -6,9 +6,11 @@ import Foundation
 import Flutter
 import UIKit
 import CoreMotion
+import CoreLocation
```

Comment:
Do we really need it here? CMAltimeter is a part of CoreMotion, which is already imported.

Reply:
You are right. My bad. Haven't got my macbook with me right now so haven't been able to test this out properly for iOS. I'll make the changes.

Reply:
Done